internal/controller/nats/integrationtests/validation/integration_test.go

```diff
@@ -267,6 +267,36 @@ func Test_Validate_UpdateNATS(t *testing.T) {
 			},
 			wantErrMsg: "fileStorage is immutable once it was set",
 		},
+		{
+			name: `validation of cluster fails, if cluster.size>1 gets set to 1'`,
```

Comment:
This sentence is a little confusing. How about:

```suggestion
			name: `validation of cluster fails, if cluster.size was set to a value >1 and now gets changed to 1'`,
```

...or something similar.

Reply:
Done!